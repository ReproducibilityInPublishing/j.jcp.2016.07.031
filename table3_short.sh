echo "AIMGM" > table3.csv

./example2 -l 9 --alpha 0.16666667 -N 32 2>/dev/null >> table3.csv
./example2 -l 9 --alpha 0.16666667 -N 64 2>/dev/null >> table3.csv
./example2 -l 9 --alpha 0.16666667 -N 128 2>/dev/null >> table3.csv
./example2 -l 9 --alpha 0.16666667 -N 256 2>/dev/null >> table3.csv
#./example2 -l 9 --alpha 0.16666667 -N 512 2>/dev/null >> table3.csv

./example2 -l 9 --alpha 0.5 -N 50 2>/dev/null >> table3.csv
./example2 -l 9 --alpha 0.5 -N 100 2>/dev/null >> table3.csv
./example2 -l 9 --alpha 0.5 -N 200 2>/dev/null >> table3.csv
./example2 -l 9 --alpha 0.5 -N 400 2>/dev/null >> table3.csv
#./example2 -l 9 --alpha 0.5 -N 800 2>/dev/null >> table3.csv

./example2 -l 9 --alpha 0.99 -N 50 2>/dev/null >> table3.csv
./example2 -l 9 --alpha 0.99 -N 100 2>/dev/null >> table3.csv
./example2 -l 9 --alpha 0.99 -N 200 2>/dev/null >> table3.csv
./example2 -l 9 --alpha 0.99 -N 400 2>/dev/null >> table3.csv
#./example2 -l 9 --alpha 0.99 -N 800 2>/dev/null >> table3.csv

echo "BFSMGM" >> table3.csv

./example2_BFSMGM -l 9 --alpha 0.16666667 -N 32 2>/dev/null >> table3.csv
./example2_BFSMGM -l 9 --alpha 0.16666667 -N 64 2>/dev/null >> table3.csv
./example2_BFSMGM -l 9 --alpha 0.16666667 -N 128 2>/dev/null >> table3.csv
./example2_BFSMGM -l 9 --alpha 0.16666667 -N 256 2>/dev/null >> table3.csv
#./example2_BFSMGM -l 9 --alpha 0.16666667 -N 512 2>/dev/null >> table3.csv

./example2_BFSMGM -l 9 --alpha 0.5 -N 50 2>/dev/null >> table3.csv
./example2_BFSMGM -l 9 --alpha 0.5 -N 100 2>/dev/null >> table3.csv
./example2_BFSMGM -l 9 --alpha 0.5 -N 200 2>/dev/null >> table3.csv
./example2_BFSMGM -l 9 --alpha 0.5 -N 400 2>/dev/null >> table3.csv
#./example2_BFSMGM -l 9 --alpha 0.5 -N 800 2>/dev/null >> table3.csv

./example2_BFSMGM -l 9 --alpha 0.99 -N 50 2>/dev/null >> table3.csv
./example2_BFSMGM -l 9 --alpha 0.99 -N 100 2>/dev/null >> table3.csv
./example2_BFSMGM -l 9 --alpha 0.99 -N 200 2>/dev/null >> table3.csv
./example2_BFSMGM -l 9 --alpha 0.99 -N 400 2>/dev/null >> table3.csv
#./example2_BFSMGM -l 9 --alpha 0.99 -N 800 2>/dev/null >> table3.csv
