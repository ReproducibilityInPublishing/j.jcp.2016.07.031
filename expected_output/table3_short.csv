AIMGM
-1,-1,1,1,0.5,5e-09,1e-08,512,0.166667,32,4,5.71002,2.610673e-05
-1,-1,1,1,0.5,5e-09,1e-08,512,0.166667,64,4,11.2265,7.866655e-06
-1,-1,1,1,0.5,5e-09,1e-08,512,0.5,50,4,8.78771,1.596971e-04
-1,-1,1,1,0.5,5e-09,1e-08,512,0.5,100,4,17.2168,5.754918e-05
-1,-1,1,1,0.5,5e-09,1e-08,512,0.99,50,4,8.76765,3.258143e-03
-1,-1,1,1,0.5,5e-09,1e-08,512,0.99,100,4,17.2502,1.623977e-03
BFSMGM
-1,-1,1,1,0.5,5e-09,1e-08,512,0.166667,32,4,9.471,2.610490e-05
-1,-1,1,1,0.5,5e-09,1e-08,512,0.166667,64,4,19.4279,7.873482e-06
-1,-1,1,1,0.5,5e-09,1e-08,512,0.5,50,4,14.9836,1.597001e-04
-1,-1,1,1,0.5,5e-09,1e-08,512,0.5,100,4,30.9812,5.754671e-05
-1,-1,1,1,0.5,5e-09,1e-08,512,0.99,50,4,14.9684,3.257906e-03
-1,-1,1,1,0.5,5e-09,1e-08,512,0.99,100,4,30.9548,1.623077e-03
